CHANGELOG.md

```diff
@@ -3,6 +3,9 @@
 Mapbox welcomes participation and contributions from everyone.
 # main
 
+## Bug fixes 🐞
```

Comment:
this will be the part of upcoming rc - could prepare the header for it already? otherwise if we land some fix that we _don't_ want to cherry-pick - they will get shuffled under the `main` section

Reply:
Please check [5e38872](https://github.com/mapbox/mapbox-maps-android/pull/1716/commits/5e388728e870de7f86b8f0affb94b3a3489165f3). Not sure if that's what it's needed.